pom.xml

```diff
@@ -342,6 +348,18 @@ THE SOFTWARE.
               <includeGroupIds>org.jenkins-ci</includeGroupIds>
             </configuration>
           </execution>
+          <execution>
+            <id>bundle-tyrus</id>
```

Comment:
Perhaps this should be using the Shade plugin, to avoid leaking JSR classes into the classpath of everything using `remoting`.